storage/storage.go

```diff
@@ -1,31 +1,36 @@
 package storage
 
-import "github.com/netlify/gotrue/models"
+import (
+	"github.com/netlify/gotrue/models"
+	uuid "github.com/satori/go.uuid"
+)
 
 // Connection is the interface a storage provider must implement.
 type Connection interface {
```

Comment:
we could cut this whole interface now that we're not supporting other than mysql interfaces right?

Reply:
We could, but it does provide a nice decoupling point.

Reply:
sweet - if you think it helps, I'm fine with leaving it.